specification/graphrbac/data-plane/stable/1.6/graphrbac.json

```diff
@@ -2465,7 +2465,6 @@
         },
         "customKeyIdentifier": {
           "type":"string",
-          "format": "byte",
```

Comment:
Cert based id should be a plain string with the thumbprint as the value

Reply:
> byte – base64-encoded characters, for example, U3dhZ2dlciByb2Nrcw==

So it depends only if you have base64 string on the wire. If yes, the right value is "byte". If no, the right value is "string"

Reply:
After check, this string is actually not byte on the wire and the fix is right.